cdins/vue-json-editor-portal
# SD-7641 Make this vue 3 compatible

## Description 
Make the vue-json-editor-portal package (https://github.com/danieljosephbrennan/vue-json-editor) vue 3 compatible

## Testing 
Clone this repo, then run these steps: 

1. Change directores to this repo `cd path/to/your-forked-package/vue-json-editor-portal`
2.  Go to this branch, then run `npm install`
3. Run `npm link`
4. Change directories to the poratal repo 
5. Delete the node_modules folder
6. Run `npm link vue-json-editor-portal`
7. Run `npm run serve` to run the portal locally
8. Make sure that all instances of the JsonEditor component are working properly

## Checklist



- [x] Branch name and commit messages match any of the following formats {issue#}-{short-description}
- [ ] New code has test coverage
- [ ] Changes to code has updated test coverage
- [ ] Documentation is updated to reflect repository changes
- [ ] Code has passed linting
- [ ] Distribution and documentation of new environment parameters
